src/core/ConfigManager.cpp

```diff
@@ -347,18 +347,8 @@ const QString & ConfigManager::value(const QString & cls,
 			}
 		}
 	}
-	static QString empty;
-	return empty;
-}
-
 
-
-const QString & ConfigManager::value(const QString & cls,
-				      const QString & attribute,
-				      const QString & defaultVal) const
-{
-	const QString & val = value(cls, attribute);
-	return val.isEmpty() ? defaultVal : val;
+	return val.value_or(defaultVal);
```

Comment:
`val` is never set, so this will always be `defaultVal`?

Reply:
> `val` is never set, so this will always be `defaultVal`?

Thanks for pointing this out. It should be fixed now.